src/widget/padding.rs

```diff
@@ -40,6 +41,33 @@ impl<T: Data> Padding<T> {
             child: WidgetPod::new(child).boxed(),
         }
     }
+
+    /// Create a new widget with the specified padding. This can either be an instance
+    /// of [`kurbo::Insets`], or it can be a tuple of floats.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use druid::widget::{Label, Padding};
+    /// use druid::kurbo::Insets;
+    ///
+    /// let child: Label<()> = Label::new("I need my space!");
+    /// let padding = Padding::with_insets((10.0, 20.0), child);
+    /// // equivalent:
+    /// let moar_padding = Padding::with_insets(Insets::uniform_xy(10.0, 20.0), padding);
+    /// ```
+    ///
+    /// [`kurbo::Insets`]: https://docs.rs/kurbo/0.5.3/kurbo/struct.Insets.html
+    pub fn with_insets(insets: impl Into<Insets>, child: impl Widget<T> + 'static) -> Padding<T> {
```

Comment:
This makes obvious sense. I'd even be ok with this being called `new`. It should work with a scalar as well, because of the into conversion, no? That's one of the things that makes me think this could be the designated constructor of choice.

Reply:
Yea, I think that makes sense. Will update.